src/scheduler/workers/analytics/main.ts

```diff
@@ -0,0 +1,24 @@
+import { schedulerLogger } from "scheduler/logger";
+import {
+  allproviderstatus,
+  ALLwhereproviderrequired,
```

Comment:
Ha, interesting spelling. Is there a reason for it?

Reply:
yeah that's my bad. I should've changed it when I was creating the view in the db, but I just copy + pasted the query that was given to me